### PR TITLE
allow database name with dot

### PR DESCRIPTION
**Pull request by Lokutos**

### CODE REVIEW

File: plugins/mysql/mysql_size_all [hunk 1]
```diff
@@ -111,6 +111,7 @@ foreach my $db (getDBList()) {
 
     $total_size = $datas+$indexes;
   }
+  $db =~ s/\./-/g;
```
Comment (sumpfralle) [inline plugins/mysql/mysql_size_all]: The dash is not an allowed character, too. Maybe it works, but by specification it is forbidden. It surely causes problems if used in `cdef` evaluation.

Instead you should do the following:
* add `use Munin::Plugin;` at the top of the script
* replace all illegal characters: `$db = clean_fieldname($db);`

(I hope, I guessed the purpose of your change correctly)

Comment (Lokutos) [reply plugins/mysql/mysql_size_all]: Hi, Thanks for reply.
Didn't insert the dash but i will check and change the lines from the original developer too.

Comment (sumpfralle) [reply plugins/mysql/mysql_size_all]: Indeed I meant `hyphen` instead of `dash`. Thanks for fixing this issue!

**Comment by sumpfralle** [comment]
Did you already take a look? Maybe introduce `clean_fieldname` instead of manual repair?

**Comment by sumpfralle** [comment]
@Lokutos: ping?

**Comment by sumpfralle** [comment]
I created a separate commit (e641c8028469e940578c460060a23a9eeb27657e) with changes similar to the ones suggested by you.
Thanks!

**Pull request closed without merging**